docs/getting-started/getting-started.dockerfile

```diff
@@ -8,7 +8,6 @@ RUN apt-get update -y && apt-get install -y \
 	python3.5 \
 	python3-pip \
 	python-setuptools \
-	ipython \
 	ipython-notebook \
```

Comment:
should `ipython-notebook` be moved as well?

Reply:
sorry for the late reply. That may be required. let me fix

Reply:
just fixed @jovfer